DirectProgramming/C++SYCL_FPGA/Tutorials/GettingStarted/fpga_compile/README.md

```diff
@@ -252,61 +289,65 @@ Notice that whether you target the FPGA emulator, FPGA simulator or FPGA hardwar
 
 ## Running the Sample
 
-1. Run the sample on the FPGA emulator (the kernel executes on the CPU):
-  ```
-  ./fpga_compile.fpga_emu     (Linux)
-  fpga_compile.fpga_emu.exe   (Windows)
-  ```
-2. Run the sample on the FPGA simulator device (the kernel executes in the simulator):
-  * On Linux
-    ```bash
-    CL_CONTEXT_MPSIM_DEVICE_INTELFPGA=1 ./fpga_compile.fpga_sim
-    ```
-  * On Windows
-    ```bash
-    set CL_CONTEXT_MPSIM_DEVICE_INTELFPGA=1
-    fpga_compile.fpga_sim.exe
-    set CL_CONTEXT_MPSIM_DEVICE_INTELFPGA=
-    ```
+ 1. Run the sample on the FPGA emulator (the kernel executes on the CPU):
+     ```
+     ./vector_add.fpga_emu     (Linux)
+     vector_add.fpga_emu.exe   (Windows)
+     ```
+2. Run the sample on the FPGA simulator device (the kernel executes in a simulator):
+   * On Linux
+      ```
+      CL_CONTEXT_MPSIM_DEVICE_INTELFPGA=1 ./vector_add.fpga_sim
+      ```
+   * On Windows
+      ```
+      set CL_CONTEXT_MPSIM_DEVICE_INTELFPGA=1
+      vector_add.fpga_sim.exe
+      set CL_CONTEXT_MPSIM_DEVICE_INTELFPGA=
+      ```
 3. Run the sample on the FPGA device (only if you ran `cmake` with `-DFPGA_DEVICE=<board-support-package>:<board-variant>`):
-  ```
-  ./fpga_compile.fpga         (Linux)
-  fpga_compile.fpga.exe       (Windows)
-  ```
+     ```
+     ./vector_add.fpga         (Linux)
+     vector_add.fpga.exe       (Windows)
+     ```
 
 ### Example of Output
 ```
-PASSED: results are correct
+using FPGA Simulator.
+add two vectors of size 256
+PASSED
 ```
 
 ## License
+Code samples are licensed under the MIT license. See
+[License.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt) for details.
 
-Code samples are licensed under the MIT license. See [License.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt) for details.
+Third party program Licenses can be found here: [third-party-programs.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/third-party-programs.txt).
 
-Third-party program Licenses can be found here: [third-party-programs.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/third-party-programs.txt).
+[1]: <https://software.intel.com/content/www/us/en/develop/documentation/oneapi-fpga-optimization-guide/top/analyze-your-design.html>
+[2]: <https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/programming-interface/fpga-flow/fpga-compilation-flags.html>
+[3]: <https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-linux-or-macos.html>
+[4]: <https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-windows.html>
```

Comment:
Insert the links inline rather than at the end

Reply:
i thought we liked to have a list of links at the end of code samples?

Reply:
I'm not aware of any samples doing so - maybe I missed something

Reply:
Change link style back to inline